src/test/java/com/ericsson/commonlibrary/proxy/JavaBeanWithoutSettersProxyTest.java

```diff
@@ -141,21 +141,5 @@ private static void invokeSetTrue(Object proxyBean, boolean bool) {
         } catch (Exception e) {
             e.printStackTrace();
         }
-    }
-
```

Comment:
Have to move these 3 classes to same package as JavaBeanWithoutSettersProxyTest.java since Javassist require this on Java11